### PR TITLE
Katello 4.0.0-rc3 updates

### PR DESCRIPTION
**Pull request by jjeffers**

### CODE REVIEW

File: configs/katello/4.0.yaml [hunk 1]
```diff
@@ -38,6 +38,9 @@
     :repo: https://github.com/theforeman/foreman_virt_who_configure.git
 :ignores:
   - 31308 # already in
+  - 31967 # for katello 3.18
+  - 31636 # migration fixes for katello 3.18
+  - 31712 # not yet included in Foreman 2.4-stable
```
Comment (jturel) [inline configs/katello/4.0.yaml:43]: Maybe this should be triaged to 4.1? The foreman issue isn't triaged for 2.4 -> https://projects.theforeman.org/issues/31629

Doing this implies you can remove the ignore here

Comment (jjeffers) [reply configs/katello/4.0.yaml]: @jturel I somehow missed this. I can update the issue, if need be.

**Comment by jturel** [comment]
3.18.3 has been added in Redmine since this was opened - that should be among the priors too!

File: configs/katello/4.0.yaml (hunk 1 above)
Comment (jturel) [reply configs/katello/4.0.yaml]: Yeah I think it should be 4.1

**Review by jturel — APPROVED** [review]

**Pull request merged**